content/topics/gdpr-ethical-review-glossary.md

```diff
@@ -114,7 +114,7 @@ Personal data processing must be protected by technical and other measures, depe
 
 -   https://www.imy.se/en/organisations/data-protection/this-applies-accordning-to-gdpr/information-security/
 
-### **lawful gound, legal basis**
+### **lawful ground, legal basis**
```

Comment:
```suggestion
### **lawful grounds, legal basis**
```

Reply:
I agree with you that "Lawful grounds" is the most appropriate!